msal/src/Microsoft.Identity.Client/IPublicClientApplication.cs

```diff
@@ -46,7 +46,7 @@ public interface IPublicClientApplication : IClientApplicationBase
 
         // expose the interactive API without UIParent only for platforms that 
         // do not need it to operate like desktop, UWP, iOS.
-#if !ANDROID
+#if !ANDROID && !FACADE
```

Comment:
Looks like FACADE is only defined for netstandard1.1; The system automatically defines the compiler variables NETSANDARD1_1 and NETSTANDARD1_3 which are used here and there throughout the code. I think FACADE should be defined for both. Let me know if you want me to pick this up.

Reply:
I agree FACADE should be defined for both. This would simplify things a lot.